### PR TITLE
Create scheduled job for recurring subscription payments #42

### LINKED ISSUE #42: Create scheduled job for recurring subscription payments
In the subscription service create methods for withdrawing money for periodic subscriptions.

The business logic of scheduled method:
1. Find all subscriptions with any recurring periodicity (i.g. WEEKLY) in the database.
2. Iterate on subscriptions and withdraw money from an account that is mentioned in the subscription.
3. Use `@Scheduled` annotation with [Cron Expressions](https://www.baeldung.com/spring-scheduled-tasks#schedule-a-task-using-cron-expressions) for making the method executing regularly.
4. For testing purposes set up execution 1 minute for WEEKLY and 5 minutes for MONTHLY subscriptions.  

Useful link:
https://www.baeldung.com/spring-scheduled-tasks

### PR DESCRIPTION
**Pull request by andy-sam**
Fixes #42

### CODE REVIEW

File: src/main/java/com/codesarge/banking/entities/SubscriptionDto.java [hunk 1]
```diff
@@ -11,14 +12,18 @@
 @NoArgsConstructor
 public class SubscriptionDto {
     private Long id;
-    private BankService bankService;
+    private String bankServiceName;
+    private Periodicity bankServicePeriodicity;
+    private Long bankServicePrice;
```
Comment (alex-anakin) [inline src/main/java/com/codesarge/banking/entities/SubscriptionDto.java:17]: we don't need this. please don't change `SubscriptionDto`

File: src/main/java/com/codesarge/banking/service/SubscriptionService.java [hunk 2]
```diff
@@ -46,4 +50,19 @@ public SubscriptionDto makeSubscription(SubscriptionRequestDto subscriptionReque
         accountService.withdrawMoney(account, bankService.getPrice());
         return savedSubscriptionDto;
     }
+
+    public List<SubscriptionDto> findSubscriptionWithPeriodicity(Periodicity periodicity) {
```
Comment (alex-anakin) [inline src/main/java/com/codesarge/banking/service/SubscriptionService.java:54]: the method is unused. please remove
P.S. Also, it's incorrectly implemented.

**Review by alex-anakin — CHANGES_REQUESTED** [review]

File: src/main/java/com/codesarge/banking/service/SubscriptionService.java (hunk 2 above)
Comment (andy-sam) [reply src/main/java/com/codesarge/banking/service/SubscriptionService.java:54]: done

File: src/main/java/com/codesarge/banking/entities/SubscriptionDto.java [hunk 3]
```diff
@@ -11,13 +11,11 @@
 @NoArgsConstructor
 public class SubscriptionDto {
     private Long id;
-    private BankService bankService;
```
Comment (alex-anakin) [inline src/main/java/com/codesarge/banking/entities/SubscriptionDto.java:14]: I asked to not change this file. Why did you remove the `bankService` field?

**Review by alex-anakin — CHANGES_REQUESTED** [review]

File: src/main/java/com/codesarge/banking/entities/SubscriptionDto.java (hunk 3 above)
Comment (andy-sam) [reply src/main/java/com/codesarge/banking/entities/SubscriptionDto.java:14]: done

**Review by alex-anakin — APPROVED** [review]
